<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogPlanNewAtack</class>
 <widget class="QDialog" name="dialogPlanNewAtack">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>457</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Planuire atac</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/atack_icon.jpg</normaloff>:/images/atack_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
font-size: 13px;
border-color: #339;
border-style: solid;
border-radius: 7;
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameVillages">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="labelMaxVillages">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Maxim 300 de sate</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelYourVillages">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>Satele tale</string>
        </property>
        <property name="textFormat">
         <enum>Qt::LogText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
        <property name="buddy">
         <cstring>&lt;h2&gt;&lt;font color=blue&gt;Satele tale &lt;/font&gt;&lt;/h2&gt;</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelEnemyVillages">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>Satele inamicului</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="plainTextEditYourVillages">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;</string>
        </property>
        <property name="tabChangesFocus">
         <bool>false</bool>
        </property>
        <property name="plainText">
         <string>Textul trebuie sa contina coordonatele in format XXX|YYY.Puteti copia de la profil,din privirea generala,etc.</string>
        </property>
        <property name="overwriteMode">
         <bool>false</bool>
        </property>
        <property name="maximumBlockCount">
         <number>300</number>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
        <property name="centerOnScroll">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPlainTextEdit" name="plainTextEditEnemyVillages">
        <property name="styleSheet">
         <string notr="true">padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;</string>
        </property>
        <property name="plainText">
         <string>Satele tinta</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="MaxEnemyTargets">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>Maxim 300/nr.de at</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAtackInfo">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelAttacksPerVillage">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Atacuri pe sat:</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelAtackLanding">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Data la care sa ajunga atacurile</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditAtackLanding">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2012</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2012</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>2050</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxSlowestUnit">
        <item>
         <property name="text">
          <string>Nobil</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unit_nobil.png</normaloff>:/images/unit_nobil.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Berbec</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unit_ram.png</normaloff>:/images/unit_ram.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Luptator cu spada</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unit_sword.png</normaloff>:/images/unit_sword.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Luptator cu toporul</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unit_axe.png</normaloff>:/images/unit_axe.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cavalerie grea</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unit_heavy.png</normaloff>:/images/unit_heavy.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cavalerie usoara</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unit_light.png</normaloff>:/images/unit_light.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spion</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unit_spy.png</normaloff>:/images/unit_spy.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSlowestUnit">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Unitatea cea mai lenta:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButtonStartPlanningAtack">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Planuieste atac</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/Attack_icon.png</normaloff>:/images/Attack_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>19</width>
          <height>19</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Nume grupa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditGroupName">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string>Numele grupei</string>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Anuleaza</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxAtacksPerVill">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="specialValueText">
         <string/>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>plainTextEditEnemyVillages</tabstop>
  <tabstop>plainTextEditYourVillages</tabstop>
  <tabstop>comboBoxSlowestUnit</tabstop>
  <tabstop>dateTimeEditAtackLanding</tabstop>
  <tabstop>pushButtonStartPlanningAtack</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>dialogPlanNewAtack</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
